### PR TITLE
added support for Apple Watch

### PR DESCRIPTION
**Pull request by oskarn97**

### CODE REVIEW

File: lib/fastlane/plugin/appicon/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Appicon
-    VERSION = "0.9.0"
+    VERSION = "0.9.1"
```
Comment (KrauseFx) [inline lib/fastlane/plugin/appicon/version.rb]: Please don't add a version bump as part of a feature PR, this should be separate 👍

**Review by KrauseFx — COMMENTED** [review]
Nice, thanks for your contribution @oskarn97, this looks great. Have you tested those changes locally and ensured that things work as expected?

File: lib/fastlane/plugin/appicon/version.rb (hunk 1 above)
Comment (oskarn97) [reply lib/fastlane/plugin/appicon/version.rb]: ok

**Comment by oskarn97** [comment]
@KrauseFx Yes, we are already using it in our production environment on multiple apps.

**Review by KrauseFx — APPROVED** [review]
Beautiful, thanks for your contribution ❤️

**Comment by oskarn97** [comment]
Thanks for merging!

**Comment by KrauseFx** [comment]
Yeah, new release is live :)

**Pull request merged**